client/components/navigation-header/index.tsx

```diff
@@ -35,6 +35,13 @@ interface Props {
 	loggedIn?: boolean;
 }
 
+// This function checks if the URL contains the 'options' query parameter and if it includes 'noCrumbs'.
+// Expired eCommerce trials use this to hide the breadcrumbs since those users can't access settings and other pages exposed in the breadcrumbs.
+const showBreadcrumb = () => {
```

Comment:
My initial approach was to include this in the controller and then prop drill all the way here. For some reason, it doesn't work when navigating.